README.md

```diff
@@ -36,5 +44,3 @@ Run the app on emulator or hardware device from Android Studio (**shift+F10** on
 If you see a home page with a login button, then things are working! Clicking the Login button will redirect you to the Okta sign-in page.
 
 You can sign in with the same account that you created when signing up for your developer org, or you can use a known username and password from your Okta Directory.
-
-> **Note:** For the sake of simplicity everything is done on the main application activity in this sample. In a real application please follow the latest [Android app architecture guide](https://developer.android.com/topic/architecture) and use Fragments and view models.
```

Comment:
is the new app following all best proactices?

Reply:
For the intent of the sample, everything has been updated in such a way that it doesn't need called out. Every app architecture will be different, and someone might not follow this guide exactly in their app. But that being said, there are no major issues with the way it's implemented (memory leaks, issues with configuration change/rotation, etc).